packages/shared.net.errors/src/UnknownError.ts

```diff
@@ -18,8 +18,8 @@ import { PalantirApiError } from "./PalantirApiError.js";
 
 export class UnknownError extends PalantirApiError {
   originalError: Error | undefined;
-  constructor(message: string, errorType: string, originalError?: Error) {
-    super(message, errorType);
+  constructor(message: string, statusCode?: number, originalError?: Error) {
```

Comment:
this would be a semver break. You would need to add it as a 4th param thats optional

Reply:
Done, I changed the second parameter (errorName renamed to match PalantirApiError and the actual field name in the API errors) to be optional which I think should preserve compatibility for the existing packages and no longer use it in the current code